lib/src/st2110/pipeline/st20_pipeline_rx.c

```diff
@@ -936,6 +936,12 @@ st20p_rx_handle st20p_rx_create(mtl_handle mt, struct st20p_rx_ops* ops) {
   size_t dst_size = 0;
   bool auto_detect = ops->flags & ST20P_RX_FLAG_AUTO_DETECT ? true : false;
```

Comment:
again, dereferencing before checking for nullptr

Reply:
Yes good catch fixed